js/movies.js

```diff
@@ -0,0 +1,46 @@
+const movies = [
+    {
+        image: "img/movie_photo/godzilla.jpg",
+        header: "Godzilla vs. Kong (2021)"
+    },
+    {
+        image: "img/movie_photo/house-of-gucci.jpg",
+        header: "House of Gucci (2021)"
+    },
+    {
+        image: "img/movie_photo/tom-and-jerry.jpg",
+        header: "Toma and Jerry (2021)"
+    },
+    {
+        image: "img/movie_photo/without-remorse.jpg",
+        header: "Without Remorse (2021)"
+    },
+]
+
+function moviePoster(URL,header) {
+    return `
+    <div class="movie-poster">
+        <img class="movie-photo" src="${URL}">
+        <button onclick="window.location.href='ticket.html'" class="ticket_">Get Tickets</button>
+        <h2 class="movie-header">"${header}"</h2>
+    </div>
+    `
+}
+ let postersData = [];
+ for(let i=0;i<movies.length;i++){
```

Comment:
try to add spacings for more readability:
```suggestion
 for(let i = 0; i < movies.length; i++){
```